internal/providers/terraform/azure/kubernetes_cluster.go

```diff
@@ -57,6 +58,46 @@ func NewAzureRMKubernetesCluster(d *schema.ResourceData, u *schema.UsageData) *s
 		aksClusterNodePool("default_node_pool", region, d.Get("default_node_pool.0"), nodeCount, u),
 	}
 
+	if d.Get("network_profile.0.load_balancer_sku").Type != gjson.Null {
+		if strings.ToLower(d.Get("network_profile.0.load_balancer_sku").String()) == "standard" {
+			if strings.Contains(strings.ToLower(region), "usgov") {
+				region = "US Gov"
+			} else if strings.Contains(strings.ToLower(region), "china") {
+				region = "Сhina"
+			} else {
+				region = "Global"
+			}
```

Comment:
I think it would be nice if you could move that logic to [this](https://github.com/infracost/infracost/blob/master/internal/providers/terraform/azure/util.go) file, as it using for several Azure resources

Reply:
I'll do that in a separate PR after this is merged. Is everything else ok with this PR? costs match?

Reply:
LGTM. I checked the costs, they are correct